packages/fx-core/src/plugins/solution/fx-solution/solution.ts

```diff
@@ -1331,6 +1303,68 @@ export class TeamsAppSolution implements Solution {
         this.AppStudioPlugin,
       ]);
 
+      const grantPermissionWithCtx: LifecyclesWithContext[] = pluginsWithCtx.map(
+        ([plugin, context]) => {
+          return [plugin?.grantPermission?.bind(plugin), context, plugin.name];
+        }
+      );
+
+      const results = await executeConcurrently("", grantPermissionWithCtx);
+      const permissions: ResourcePermission[] = [];
+      for (const result of results) {
+        if (result.isErr()) {
+          return result;
```

Comment:
Here shall we just return? According to yanliang's doc, we need to print some info.